pkg/controllers/provisioning/scheduling/node.go

```diff
@@ -27,65 +29,103 @@ import (
 // Node is a set of constraints, compatible pods, and possible instance types that could fulfill these constraints. This
 // will be turned into one or more actual node instances within the cluster after bin packing.
 type Node struct {
-	Constraints         *v1alpha5.Constraints
+	v1alpha5.Constraints
 	InstanceTypeOptions []cloudprovider.InstanceType
 	Pods                []*v1.Pod
-	unschedulable       bool // if true, the node will accept no more pods
+
+	podResources    v1.ResourceList
+	daemonResources v1.ResourceList
```

Comment:
Since daemonResources are now static, can we combine them with podResources and never need to think of them again, perhaps renaming to something like `node.reserved`.